docs/explore/qrng/index.md

```diff
@@ -121,10 +88,4 @@ cheaper, and more customizable request fulfillments. In addition, integrations
 to more chains and QRNG providers will be made available over time, and
 higher-level QRNG services will be built leveraging these.
 
-See these guides and learn how to use QRNG:
```

Comment:
Why not link to the QRNG [getting started guide page](https://docs.api3.org/guides/qrng/)? It seems like we'd want to have an actionable next step after viewing this Explore page.